src/NServiceBus.Metrics/ISignalProbe.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus
+{
+    using System;
+
+    /// <summary>
+    /// Probe that signals event occurrence
+    /// </summary>
+    public interface ISignalProbe : IProbe
+    {
+        /// <summary>
+        /// Enables registering action called on event occurrence.
+        /// </summary>
+        void Register(Action observer);
```

Comment:
I think with this description it is a bit clearer. It is just an event that occurred without any payload or meaningful measurement. The event name is the signal right?